bug: NameError: name 'CellInfo' is not defined when using CellInfo at top level

### Did you check docs and existing issues?

- [x] I have read all the docs
- [x] I have updated to the latest version before submitting this issue
- [x] I have searched the existing issues

### Affected Component

Aegis

### Operating system/version

Ubuntu 24

### Describe the bug

Using `CellInfo` type at the top level (outside of a function) causes an error. I don't think this is intended behaviour because the same is not true for `Location`. Not critical but it makes it hard to write type-safe code. This may exist for other types too, I haven't checked them all.

### Steps To Reproduce

main.py:
```python
from aegis_game.stub import CellInfo, Location

loc: Location  # this is fine
info: CellInfo  # this causes "NameError: name 'CellInfo' is not defined"


def think() -> None:
    pass


def foo() -> None:
    information: CellInfo # this is fine
```

Run simulation with `aegis launch`

Produces:

```
[Agent] [Agent#(10578:GOOBS)@1] Traceback (most recent call last):
  File "/home/tim/uni/CPSC383/A2_2/.venv/lib/python3.13/site-packages/_aegis_game/sandbox/core.py", line 167, in init
    exec(self.code["main"], self.namespace)  # noqa: S102
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "main.py", line 3, in <module>
NameError: name 'CellInfo' is not defined

[Agent] [Agent#(10578:GOOBS)@1] Think doesn't exist
[Agent] [Agent#(10578:GOOBS)@1] [ERROR] penalized -10 energy for error.
```

Side note: the blamed line in any traceback (line 3 in this case) is always off (short) by 1.

### Expected Behavior

No errors.

--- a/src/_aegis_game/game.py
+++ b/src/_aegis_game/game.py
@@ -504,10 +504,12 @@ class Game:
     def methods(self, ac: AgentController) -> MethodDict:
         return {
             "AgentType": AgentType,
+            "CellInfo": CellInfo,
             "Direction": Direction,
             "Location": Location,
             "Rubble": Rubble,
             "Survivor": Survivor,
+            "Team": Team,
             "get_round_number": ac.get_round_number,
             "get_id": ac.get_id,
             "get_type": ac.get_type,

